<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_import">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>370</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Cantarell&quot;;</string>
   </property>
   <property name="text">
    <string>Import Template</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_export">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Cantarell&quot;;</string>
   </property>
   <property name="text">
    <string>Export Script</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>261</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Cantarell&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_templateName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Cantarell&quot;;</string>
   </property>
   <property name="text">
    <string>Set template name here</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_importScript">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Cantarell&quot;;</string>
   </property>
   <property name="text">
    <string>Import Script</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
